data/mods/Dark-Skies-Above/factions.json

```
@@ -2,38 +2,47 @@
   {
     "type": "MONSTER_FACTION",
     "name": "invader_alien",
-    "neutral": [ "small_animal", "fish", "animal", "lizard" ]
+    "base_faction": "nether",
+    "neutral": [ "alien_carnivore", "alien_herbivore", "stray", "aquatic_predator" ],
+    "by_mood": "animal"
   },
   {
     "type": "MONSTER_FACTION",
     "name": "alien_herbivore",
-    "base_faction": "small_animal"
+    "base_faction": "herbivore",
+    "neutral": [ "invader_alien", "stray" ],
+    "by_mood": [ "alien_carnivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "lizard",
-    "neutral": [ "small_animal", "invader_alien" ],
-    "by_mood": "lizard"
+    "name": "alien_carnivore",
+    "base_faction": "mutant",
+    "neutral": [ "invader_alien", "stray", "aquatic_predator" ],
+    "hate": [ "alien_herbivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "stray",
-    "//": "largely neutral to animals and things that don't live",
-    "neutral": [ "small_animal", "fish", "animal", "fox", "cat", "dog", "invader_alien" ]
+    "name": "alien_packliz",
+    "base_faction": "alien_carnivore",
+    "by_mood": "alien_carnivore"
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "lurker",
-    "//": "TODO assign base faction and/or attitude"
+    "name": "stray",
+    "//": "largely neutral to animals, but will retaliate.  strictly neutral to their creators.",
+    "base_faction": "bot",
+    "neutral": [ "invader_alien", "alien_herbivore", "alien_carnivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "packliz",
-    "//": "TODO assign base faction and/or attitude"
+    "name": "lurker",
+    "base_faction": "alien_carnivore",
+    "hate": [ "fish", "aquatic_predator" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "bileworm",
-    "//": "TODO assign base faction and/or attitude"
+    "base_faction": "animal",
+    "by_mood": "small_animal"
```

```suggestion
    "name": "bileworm",
    "base_faction": "dog",
    "by_mood": [ "small_animal", "dog" ]
```
Again, also needed will be a duplicate of "dog" from #48077's data/json/monster_factions.json.